hydra-node/src/Hydra/Network/Heartbeat.hs

```diff
@@ -110,8 +111,8 @@ updateStateFromOutgoingMessages ::
   TVar m HeartbeatState ->
   Network m (Heartbeat (Message msg)) ->
   Network m (Message msg)
-updateStateFromOutgoingMessages localhost heartbeatState Network{broadcast, getPeers, setPeers} =
-  Network bcast getPeers setPeers
+updateStateFromOutgoingMessages localhost heartbeatState Network{broadcast} =
+  Network bcast (pure []) (\_ -> pure ())
```

Comment:
This doesn't sound quite right :thinking:, why is this redefining the `getPeers` and `setPeers` definitions?

Reply:
my bad, fixing it.